File: java/src/main/java/fr/soat/cleancoders/exo2/MissingTimePropertiesException.java
package fr.soat.cleancoders.exo2;

public class MissingTimePropertiesException extends Exception {

    private String message;

    MissingTimePropertiesException(String message) {
        this.message = message;
    }

    @Override
    public String getMessage() {
        return message;
    }
}

File: csharp/Exo2/MissingTimePropertiesException.cs
using System;

namespace Soat.CleanCoders.Refactoring.Exo2
{
    public class MissingTimePropertiesException : Exception
    {
        public override string Message { get; }

        public MissingTimePropertiesException(string message)
        {
            Message = message;
        }
    }
}
